### DISCUSSION TITLE
Is there a way to show the correct answer during Fast Money?

### DISCUSSION THREAD

**Discussion by jonathanlaniado** [joshzcold/Cold-Family-Feud#90] [General]

**Comment by joshzcold** [comment]
Sorry I didn't see this until now.

Currently no. You would have to type in the right answer and click on "Reveal Answer" again to "reveal" it.

Points only go towards the final score when you click "Award Points". Currently you can "reveal answer" multiple times without effecting the score.

I believe the way it works on the show is that the announcer only says "[Answer]  was the number 1 answer" and moves on.
